website/content/api-docs/auth/gcp.mdx

```diff
@@ -21,7 +21,25 @@ at any location, please update your API calls accordingly.
 Configures the credentials required for the plugin to perform API calls
 to Google Cloud. These credentials will be used to query the status of IAM
 entities and get service account or other Google public certificates
-to confirm signed JWTs passed in during login.
+to confirm signed JWTs passed in during login. Credentials can be configured
+either with Application Credentials for a privileged service account,
+or using Plugin Workload Identity Federation (WIF).
+
+### IAM
+There are multiple ways to pass root credentials to the Vault server, specified
+below with the highest precedence first. If credentials already exist, this will
+overwrite them.
+
+The official Google Cloud SDK is used for sourcing credentials from env vars and shared
+files.
+
+- Static credentials provided to the API as a payload
```

Comment:
```suggestion
1. Provide static credentials to the API as a payload.
```
Style correction: write in active voice, use numbered list since these are supposed to be in order of precedence.